tasks/cockpit-tasks

```diff
@@ -21,6 +21,10 @@ if [ -r /secrets/npm-registry.crt ]; then
     export NODE_EXTRA_CA_CERTS=/secrets/npm-registry.crt
 fi
 
+# prone to timeouts and errors with lots of parallel containers
+npm config set fetch-retries 6
+npm config set fetch-timeout 600000
```

Comment:
ah, that's milliseconds, so one minute? And interesting, it's lower then the wopping 5 minutes! default.

Reply:
Right, see https://docs.npmjs.com/cli/v7/using-npm/config#fetch-timeout